### PR TITLE
Fixed filtering and sorting of the meeting listings

### PR DESCRIPTION
**Pull request by phgross**
:warning: based on #762 

I've used the the already existing `SqlTableSource` for all the meeting listings, this provides filter and sorting functionality.

@deiferni please have a look ...

### CODE REVIEW

File: docs/HISTORY.txt [hunk 1]
```diff
@@ -5,6 +5,15 @@ Changelog
 4.2.0 (unreleased)
 ------------------
 
+- Meeting listing improvements:
+    - Disable selection links for listins without checkbox column.
```
Comment (deiferni) [inline docs/HISTORY.txt]: typo listing :wink:

**Comment by deiferni** [comment]
@phgross the other PR is merged, plz rebase

**Comment by phgross** [comment]
@deiferni done ...

**Comment by deiferni** [comment]
i'm getting an error on the committee overview page. currently we don't have created/modified columns. maybe we should :wink: 
```
Traceback (innermost last):
  Module ZPublisher.Publish, line 126, in publish
  Module ZPublisher.mapply, line 77, in mapply
  Module ZPublisher.Publish, line 46, in call_object
  Module ftw.tabbedview.browser.tabbed, line 153, in listing
  Module ftw.tabbedview.browser.listing, line 109, in __call__
  Module ftw.tabbedview.browser.listing, line 197, in update
  Module opengever.tabbedview.browser.sqltablelisting, line 106, in search_results
  Module sqlalchemy.orm.query, line 2115, in all
  Module sqlalchemy.orm.query, line 2227, in __iter__
  Module sqlalchemy.orm.query, line 2242, in _execute_and_instances
  Module sqlalchemy.engine.base, line 1449, in execute
  Module sqlalchemy.engine.base, line 1584, in _execute_clauseelement
  Module sqlalchemy.engine.base, line 1698, in _execute_context
  Module sqlalchemy.engine.base, line 1691, in _execute_context
  Module sqlalchemy.engine.default, line 331, in do_execute
  Module MySQLdb.cursors, line 174, in execute
  Module MySQLdb.connections, line 36, in defaulterrorhandler
OperationalError: (OperationalError) (1054, "Unknown column 'modified' in 'order clause'") 'SELECT committees.id AS committees_id, committees.admin_unit_id AS committees_admin_unit_id, committees.int_id AS committees_int_id, committees.title AS committees_title, committees.physical_path AS committees_physical_path \nFROM committees \nWHERE committees.admin_unit_id = %s ORDER BY modified ASC' (u'fd',)
```

**Comment by phgross** [comment]
@deiferni That just happens because of an old dictstorage entry. Try to reset the table configuration with the tabbedview-menu.

This should never happen again because of the default sort_on attribute (see  https://github.com/4teamwork/opengever.core/blob/pg_meeting_listing_filter_sorting_fix/opengever/meeting/browser/committeecontainertabs.py#L13)

**Comment by deiferni** [comment]
@phgross oh - yes indeed! works flawlessly after a reset. :+1:

**Pull request merged**